config/webpack/terra-dev-site.webpack.config.js

```diff
@@ -79,25 +103,49 @@ const devSiteConfig = (env = {}, argv = {}) => {
     console.log('Generated Aliases', alias);
   }
 
+  const indexEntryPoints = [];
+  indexEntryPoints.push('terra-dev-site');
+  // Strip the trailing / from the public path.
+  const basename = publicPath.slice(0, -1);
+
   return {
     entry: {
       'terra-dev-site': path.resolve(path.join(__dirname, '..', '..', 'lib', 'Index')),
+      rewriteHistory: path.resolve(path.join(__dirname, '..', '..', 'lib', 'rewriteHistory')),
+      redirect: path.resolve(path.join(__dirname, '..', '..', 'lib', 'redirect')),
     },
     plugins: [
-      new HtmlWebpackPlugin({
+      // terra-dev-site index.html
+      indexPlugin({
         title: siteConfig.appConfig.title,
-        template: path.join(__dirname, '..', '..', 'lib', 'index.html'),
+        filename: 'index.html',
         lang,
-        dir: siteConfig.appConfig.defaultDirection,
-        favicon: siteConfig.appConfig.favicon,
-        headHtml: [getNewRelicJS()].concat(siteConfig.appConfig.headHtml),
-        inject: false, // this turns off auto injection. We handle this ourselves in the template.
+        siteConfig,
+        indexEntryPoints,
+        entry: 'terra-dev-site',
+      }),
+      // 404.html
+      new HtmlWebpackPlugin({
+        filename: '404.html',
+        template: path.join(__dirname, '..', '..', 'lib', '404.html'),
+        inject: 'head',
+        chunks: ['redirect'],
```

Comment:
We include the `'redirect'` chunks, how does this work with our polyfills, do we need to include those chunks as well or should we exclude the 'dev-site' chunk instead?

Reply:
redirect doesn't need polyfils